benchmark/scenarios/test/test_write_request_load.py

```diff
@@ -0,0 +1,362 @@
+from twisted.trial.unittest import SynchronousTestCase
+
+from itertools import repeat
+from uuid import uuid4
+from ipaddr import IPAddress
+from twisted.internet.defer import succeed, Deferred
```

Comment:
Purely stylistic comment: I'd add an empty line before this line, so that all twisted imports are in a separate block.